ipatests/test_integration/test_sssd.py

```diff
@@ -108,3 +114,57 @@ def test_auth_cache_enabled_when_configured(self, user):
             assert self.is_auth_cached(self.users[user])
             time.sleep(10)
             assert not self.is_auth_cached(self.users[user])
+
+    @contextmanager
+    def filter_user_setup(self, user, provider='ipa'):
+        tasks.kinit_admin(self.master)
+        master = self.master
+        sssd_conf_backup = tasks.FileBackup(master, paths.SSSD_CONF)
+        if user['name'] == 'fakeuser':
```

Comment:
"fakeuser" is a value passed to `test_filter_users_start_sssd_offline()`

user['name'] is never equal to 'fakeuser'